docs/help-center/docs/warden-app/ward-airdrop.md

```
@@ -0,0 +1,156 @@
+- **Distribution mechanism**: [automatic](#automatic-distribution).
+
+### OPEN OG holders & stakers
+
+- **Eligibility criteria**:
+  - Hold the minimum required amount of **LVVA** or **staked LVVA** 
+  - Hold the unique NFT on **Base**, owned by the same wallet address that held LVVA
+  - Link your wallet containing these assets to your Warden account
+  - Register for the airdrop
+- **Distribution mechanism**: [automatic](#automatic-distribution).
+
+###  Pre-Early Subscription NFT holders
```

markdownlint-cli2 (0.20.0)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)